docs/deploy-cephfs.md

```diff
@@ -27,20 +27,6 @@ make image-cephcsi
 
 ## Configuration
 
-**NOTE:** To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2
-cluster (not deployed by rook), you need to add the following settings in the
-`mgr` section of the ceph.conf used by the Ceph manager daemon, and restart the
-Ceph manager daemon.
-
-```
-[mgr]
-client mount uid = 0
-client mount gid = 0
-```
-
-This is due to an [issue](http://tracker.ceph.com/issues/40927) in Ceph v14.2.2
```

Comment:
if we are removing this we also need to drop support for 14.2.2?

Reply:
@Madhu-1 the issue fix is available since 14.2.2-x commit.  there are 2 aspects to drop support of 14.2.2. First of all, 14.x is out of support from Ceph POV. so, I am planning to have a followup PR which update the README to mention the support is from 15.x of ceph which is listed as active release. in short drop 14.x itself.

Reply:
here it is https://github.com/ceph/ceph-csi/pull/3513